lib/scarpe/wv/arrow.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Scarpe::Webview
+  class Arrow < Drawable
+    def initialize(properties)
```

Comment:
If initialize is just calling super, can probably just not define it and get the same effect.